build/ci/pr/Jenkinsfile

```diff
@@ -90,3 +104,16 @@ def notOnlyDocs() {
     	returnStatus: true
     ) == 0
 }
+
+def ifImageNeedsRebuilding() {
+    sh """
+        md5sum operators/Gopkg.lock build/ci/Dockerfile local-volume/Gopkg.lock \
```

Comment:
I wonder if we should continue to manage `local-volume` with the `operators`.

Reply:
What sounds like a big discussion topic... Maybe we can merge PR as it is and discuss our approach to `local-volume` separately?

My intentions for adding `local-volume/Gopkg.lock` was because of current status where for PR we are running both unit and integration tests for `local-volume` too. And having dependencies cached only for operator doesn't feel right in this case.